ui/base/ime/input_method_minimal_unittest.cc

```diff
@@ -0,0 +1,82 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ui/base/ime/input_method_minimal.h"
+
+#include "base/macros.h"
+#include "base/memory/singleton.h"
+#include "testing/gtest/include/gtest/gtest.h"
+#include "ui/base/ime/dummy_text_input_client.h"
+#include "ui/base/ime/input_method_delegate.h"
+#include "ui/base/ime/input_method_initializer.h"
+#include "ui/events/event.h"
+
+namespace ui {
+namespace {
+
+class InputMethodMinimalTest : public internal::InputMethodDelegate,
```

Comment:
I'd rather create a separate class, which inherits InputMethodDelegate, and another class, which inherits DummyTextInputClient, and leave InputMethodMinimalTest to be a test launcher (check input_method_auralinux_unittest.cc)

Reply:
Done.